src/main/java/org/mamute/dao/QuestionDAO.java

```diff
@@ -278,5 +277,126 @@ public Question questionWith(Attachment attachment) {
 				.setParameter("a", attachment)
 				.uniqueResult();
 	}
-}
 
+	public void delete(Question question) {
```

Comment:
Yeah, it looks like we only have a few raw queries:

```
src/integration/java/org/mamute/util/DataImport.java:           session.createSQLQuery("insert into Users (id, moderator,karma, name, slugge
src/main/java/org/mamute/dao/AnswerDAO.java:            Query query = session.createSQLQuery(sql);
src/main/java/org/mamute/dao/QuestionDAO.java:          Query query = session.createSQLQuery(sql);
src/main/migrations/org/mamute/migration/DatabaseManager.java:          session.createSQLQuery(sql).executeUpdate();
src/main/migrations/org/mamute/migration/DatabaseManager.java:                                  session.createSQLQuery(fullQuery.toString())
```

But I think that none of them needs to be updated :-)

I'll refactor the solution to use logical exclusion them.